src/ts/ConvivaAnalytics.ts

```diff
@@ -34,6 +32,61 @@ export interface ConvivaAnalyticsConfiguration {
    * user agent string parsing by the Conviva SDK. (default: WEB)
    */
   deviceCategory?: Conviva.Constants.DeviceCategory;
```

Comment:
I think we should deprecate this property here in favor our the `deviceMetadata.category`

Reply:
Hi, @stonko1994. Please confirm if should I deprecate that field and what is the process you have to do it. (Just adding a `@deprecated` JSDoc tag? Adding a warning to the console when this field is used?)

Reply:
Yes, just a `@deprecated` JSDoc tag is enough 🙁